src/MetricsUploader/CaptureMetricTest.cpp

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <chrono>
+#include <thread>
+
+#include "MetricsUploader/CaptureMetric.h"
+#include "MetricsUploader/MetricsUploader.h"
+#include "orbit_log_event.pb.h"
+
+namespace orbit_metrics_uploader {
+
+namespace {
+
+constexpr CaptureStartData kTestStartData{
+    1 /*number_of_instrumented_functions*/,
+    2 /*number_of_frame_tracks*/,
+    3 /*number_of_manual_start_timers*/,
+    4 /*number_of_manual_stop_timers*/,
+    5 /*number_of_manual_start_async_timers*/,
+    6 /*number_of_manual_stop_async_timers*/,
+    7 /*number_of_manual_tracked_values*/
+};
+
+bool CaptureDataProtoContainsCaptureStartData(const OrbitCaptureData& capture_data,
+                                              const CaptureStartData& start_data) {
+  return capture_data.number_of_instrumented_functions() ==
```

Comment:
I'm not sure if I understand. Why we are making this comparisons here?

Reply:
The function `CaptureDataProtoContainsCaptureStartData` is just a helper function to compare the input of a test to the output of that test. This is to have less duplicate code. The alternative is to do an EXPECT_EQ() for each of these.

Reply:
Ah, ok. It makes sense now. I would like to rename it, for me "Contains" it's about if we have this data more than that the data has the same value.

I would rename it to something like "HasSameCaptureStartData"

Reply:
done :)